src/etc/inc/plugins.inc.d/unbound.inc

```diff
@@ -560,7 +569,13 @@ function unbound_add_host_entries($ifconfig_details = null)
                                 $unbound_entries .= "local-zone: \"{$alias['domain']}\" redirect\n";
                                 $unbound_entries .= "local-data: \"{$alias['domain']} IN {$host->rr} {$host->server}\"\n";
                             } else {
-                                $unbound_entries .= "local-data-ptr: \"{$host->server} {$alias['hostname']}{$alias['domain']}\"\n";
+                                if (($alias === $tmp_aliases[0] || $tmp_aliases[0]['hostname'] === '*') && !in_array($host->server, $ptr_records, true)) {
```

Comment:
Perhaps log an error here when overlapping PTR exists?

Reply:
Ideally this should be handled by extending the front-end validation, a warning seems more fitting in the grand scheme of things.

Reply:
yeah, the log is cheaper here, warning is ok (still speaking of "log_error" as log error but we should be adhering to severity correctness more)

Reply:
@fichtner https://github.com/opnsense/core/pull/5925/commits/9a5d8f851f36876cdd236dd461c17fcef2efbded FYI